install-dev-dependencies.sh

```diff
@@ -4,7 +4,7 @@
 set -o errexit -o xtrace
 if type apt-get >/dev/null ; then
   sudo apt-get install libxml2-dev libxml2-utils libxslt1-dev python-dev \
-    firefox chromium-browser zip sqlite3 python-pip libcurl4-openssl-dev
+    firefox chromium-browser zip sqlite3 python-pip libcurl4-openssl-dev libssl-dev
```

Comment:
Out of interest, why do you need to add libssl-dev in?

@jsha I'm guessing you're ok with this change?